mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -97,4 +90,56 @@ if( BUILD_FAT_LIBMLIRMIOPEN )
   install(TARGETS MLIRMIOpen
     ARCHIVE DESTINATION lib)
 
+  # Install Miir.h to ${CMAKE_INSTALL_PREFIX}/include/<project-name>-<version>/
+  # as part of component libMLIRMIOpen
+  set(PACKAGE_DIR MLIRMIOpen-${CMAKE_PROJECT_VERSION})
```

Comment:
Do we want `CMAKE_PROJECT_VERSION` here? I don't think we're using that field, and I don't want to give us a trailing dash.